src/layman/rest_publications.py

```diff
@@ -0,0 +1,22 @@
+from flask import Blueprint, g, request, current_app as app
+
+from layman.authn import authenticate, get_authn_username
+from layman.authz import authorize_publications_decorator
+from layman.common import rest as rest_common
+
+bp = Blueprint('rest_publications', __name__)
+
+
+@bp.before_request
+@authenticate
+@authorize_publications_decorator
+def before_request():
+    pass
+
+
+@bp.route(f"/publications", methods=['GET'])
```

Comment:
Let's move `publications` to constant, place it in layman_settings.py near REST_USERS_PREFIX and REST_WORKSPACES_PREFIX